src/NuGetPackageVerifier/CompositeRules/SigningVerificationCompositeRule.cs

```diff
@@ -15,6 +15,7 @@ public class SigningVerificationCompositeRule : IPackageVerifierRule
             new AuthenticodeSigningRule(),
             new PowerShellScriptIsSignedRule(),
             new RequiredNuSpecInfoRule(),
+            new DefaultCompositeRule(),
```

Comment:
Ah you presumably mean the `project.json` file (which generates the various other attributes). I added it to my list of metadata to fix/add.